venues/ICLR.cc/2018/Conference/python/programchairs/get-review-info.py

```diff
@@ -0,0 +1,136 @@
+reviewers = client.get_groups(config.CONF+'/Paper.*/Reviewers')
+for paper_reviewers in reviewers:  
+    paper_number = int(paper_reviewers.id.split('Paper')[1].split('/Reviewers')[0])
+    if paper_number in paper_data:
+        for reviewer in paper_reviewers.members:
+            paper_data[paper_number]['reviewers'][reviewer] = {}
+            paper_data[paper_number]['reviewers'][reviewer]['rating'] = -1
+            paper_data[paper_number]['reviewers'][reviewer]['confidence'] = -1
+            paper_data[paper_number]['reviewer_count'] += 1
+            reviews_total += 1
+
+
+# translate anon reviewers into ids
+print "Filling in anonymous reviewers"
+anon_reviewers = client.get_groups(id = config.CONF+'/Paper.*/AnonReviewer.*')
+# reviewers[anon_name]=id
+reviewers = {}
+for anon in anon_reviewers:
+    if anon.members:
+        reviewers[anon.id]=anon.members[0]
+
+# fill in reviews per review/paper
+print "Filling in reviews"
+# can download max of 2k notes at a time
+reviews = []
+limit = 2000
+offset = 0
+notes_call_finished = False
+while not notes_call_finished:
+    notes_batch = client.get_notes(invitation='ICLR.cc/2018/Conference/-/Paper.*/Official_Review', offset=offset, limit=limit)
+    reviews += notes_batch
+    offset += limit
+    if len(notes_batch) < limit:
+        notes_call_finished = True
+
+for review in reviews:
+    paper_number = int(review.invitation.split('Paper')[1].split('/Official_Review')[0])
+    # check for review.writers in reviewers for odd case where there was a review submitted before the reviewer was removed.
+    if paper_number in paper_data and review.writers[0] in reviewers:
+        reviewer = reviewers[review.writers[0]]
+        if reviewer in paper_data[paper_number]['reviewers']:
+            paper_data[paper_number]['reviewers'][reviewer]['rating'] = review.content['rating'].split(':')[0]
+            paper_data[paper_number]['reviewers'][reviewer]['confidence'] = review.content['confidence'].split(':')[0]
+            paper_data[paper_number]['review_count'] += 1
+            reviews_complete += 1
+        else:
+            print "Error missing reviewer? "+reviewer
+
+# print to file reviewer, number of papers assigned and list of paper numbers
+with open(args.file, 'wb') as outfile:
+
+    csvwriter = csv.writer(outfile, delimiter=',')
+    # write header row
+    row = ['Paper Num', 'Title', 'Area Chair', 'reviews complete/total', 'Reviewer1', 'rating', 'confidence', 'Reviewer2', 'rating', 'confidence', 'Reviewer3', 'rating', 'confidence']
+    csvwriter.writerow(row)
+
+    for number in paper_data:
+        if args.ac == None or args.ac == paper_data[number]['AC']:
+            row = []
+            row.append(number)
+            row.append(paper_data[number]['title'].encode('UTF-8'))
+            row.append(paper_data[number]['AC'].encode('UTF-8'))
+            row.append("'{0}/{1}'".format(paper_data[number]['review_count'], paper_data[number]['reviewer_count']))
```

Comment:
instead of showing these two values in a single column. Could we split it in two? it is easier to sort in a excel:

- reviews_completed
- reviews_total